### PR TITLE
ATS-692 Fix VSCode compiler error with Tika-parsers

### PR DESCRIPTION
**Pull request by PunchedChimera**
Fixes compiler/editor error: The package javax.xml.<artifactId> is accessible from more than one module.
Currently this is not showing in java compilers due to a bug in the compiler: https://bugs.openjdk.java.net/browse/JDK-8215739
Once this bug is fixed and the error is reported correctly this will not build correctly. Excluding xml-apis:xml-apis from the classpath will fix this.

### CODE REVIEW

**Comment by PunchedChimera** [comment]
> Was this a recently introduced issue - eg. could it be fallout from recent ATS-674 ?

It appears so, this error is not present on SP/1.3N or SP/2.0.N.

File: alfresco-transform-tika/alfresco-transform-tika/pom.xml [hunk 1]
```diff
@@ -49,6 +49,10 @@
                     <groupId>org.quartz-scheduler</groupId>
                     <artifactId>quartz</artifactId>
                 </exclusion>
+                <exclusion>
```
Comment (montgolfiere) [inline alfresco-transform-tika/alfresco-transform-tika/pom.xml]: Why is this change required now (but not previously) ? Are these libraries provided elsewhere ? If not, when does Tika use these libraries at runtime ?

**Comment by montgolfiere** [comment]
Please also check 21.1 tag (in addition to SP/1.3N or SP/2.0.N).
